web/src/components/auth/SelfRegSnackbar.tsx

```diff
@@ -69,7 +76,7 @@ export const SelfRegSnackbar = (): ReactElement => {
         )}
         <div>
           {alertMap[registrationAlertStatus] === "success" ? (
-            <h3 className="padding-right-2">{getTitle()}</h3>
+            <h3 className="padding-right-2">{getSuccessTitle()}</h3>
```

Comment:
[pebble] it seems like the we are using a header for the success snackbar. We should be able to pass it into the `heading` on line 58

Reply:
I believe its a special case because of the alternate style, so it needs to be out here to look as desired. Its the very bottom custom one. 
<img width="670" alt="image" src="https://github.com/newjersey/navigator.business.nj.gov/assets/22485301/6bdf9bac-ad44-4e8c-b3ff-520542d90e43">